File: Exercicios/Python/Exercicio11.py
totalIdades = 0
mediaIdades = 0
pesoMaiorQueNoventa = 0
for x in range(7):
    totalIdades += int(input("Digite a idade de uma pessoa: "))
    peso = int(input("Digite o peso dessa pessoa: "))
    if peso > 90:
        pesoMaiorQueNoventa += 1
mediaIdades = totalIdades / 7
print("A media das idades das sete pessoas é: " + str(mediaIdades) + " e " + str(pesoMaiorQueNoventa) + " pessoas pesam mais que 90kg.")

File: Exercicios/Java/Exercicio11.java
import java.util.Scanner;

public class Exercicio11 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int totalIdades = 0, pesoMaiorQueNoventa = 0;
        float peso, mediaIdades;
        for(int i = 0; i < 7; i++){
            System.out.print("Digite a idade de uma pessoa: ");
            totalIdades += sc.nextInt();
            System.out.print("Digite o peso dessa pessoa: ");
            peso = sc.nextFloat();
            if(peso > 90){
                pesoMaiorQueNoventa++;
            }            
        }
        sc.close();
        mediaIdades = totalIdades / 7;
        System.out.println("A media das idades das sete pessoas é: " + mediaIdades + " e " + pesoMaiorQueNoventa + " pessoas pesam mais que 90kg.");
    }
}